src/main/kotlin/dev/stalla/builder/validating/ValidatingAtomBuilder.kt

```diff
@@ -33,9 +30,10 @@ internal class ValidatingAtomBuilder : AtomBuilder {
             return null
         }
 
-        val authors = authorBuilders.mapNotNull { it.build() }
-        val contributors = contributorBuilders.mapNotNull { it.build() }
-        val links = linkBuilders.mapNotNull { it.build() }
-        return Atom(authors, contributors, links)
+        return Atom(
+            authors = authorBuilders.mapNotNull { it.build() },
+            contributors = contributorBuilders.mapNotNull { it.build() },
+            links = linkBuilders.mapNotNull { it.build() }
+        )
```

Comment:
Sometimes we've built before the model construction, sometimes we did it inline. I've switched to a uniform style. Not a result of detekt though.